src/bci_build/package/appcontainers.py

```diff
@@ -484,3 +485,41 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for os_version in (OsVersion.TUMBLEWEED,)
 ]
+
+REDIS_CONTAINERS = [
+    ApplicationStackContainer(
+        name="redis",
+        pretty_name="Advanced Key-Value Store",
+        from_image=generate_from_image_tag(os_version, "bci-micro"),
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version="%%redis_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%redis_version%%",
+                package_name="redis",
+                parse_version=ParseVersion.MINOR,
+            )
+        ],
+        license="BSD-3-Clause",
+        package_list=[
+            Package(name, pkg_type=PackageType.BOOTSTRAP)
+            for name in (
+                "redis",
+                "util-linux",
+            )
+        ],
+        config_sh_script=textwrap.dedent(
+            """
+            chown redis:redis /var/lib/redis
+            """
+        ),
+        entrypoint_user="redis",
+        cmd=["redis-server", "--protected-mode no", "--dir /var/lib/redis"],
+        volumes=["/var/lib/redis"],
+        exposes_tcp=[6379],
+        build_recipe_type=BuildType.KIWI,
+    )
+    for os_version in ALL_NONBASE_OS_VERSIONS
```

Comment:
afaik redis  is dropped in SP7 and tumbleweed, so we should specify this only for SP6 for now.

Reply:
Ok. Then I am a bit late to the party.
Probably we should move to valkey soon